packages/frontend/pages/strategies.tsx

```diff
@@ -352,25 +355,54 @@ const Strategies: React.FC = () => {
                       tooltip={Tooltips.StrategyProfitThreshold}
                     />
                   ) : (
-                    <StrategyInfoItem
-                      value={
-                        '~' +
-                        lowerPriceBandForProfitability.toFixed(2) +
-                        ' - ' +
-                        upperPriceBandForProfitability.toFixed(2)
-                      }
-                      label={
-                        'Approx Profitable (' + (profitableMovePercentV2 * 100).toFixed(2) + '%) between ETH Prices ($)'
-                      }
-                      tooltip={Tooltips.StrategyProfitThreshold}
-                    />
+                    
+                    crabLoading ? (
+                      <>
+                          <StrategyInfoItem
+                          value='-'
+                          label={
```

Comment:
@DemolaJames can you bring the if condition inside label .

i.e const lowerBand = crabLoading ? "-" : lowerPriceBandForProfitability.toFixed(2)

Reply:
Refactored